content/02.main-text.md

```diff
@@ -211,6 +214,8 @@ We annotated the author list to indicate that author order was partly randomized
 
 ## Discussion
 
+### Prior works
```

Comment:
I put discussion subheadings, which was helpful for me despite being somewhat avant-garde. I don't have a strong opinion on them so happy to rename or remove.

Reply:
I don't mind the subheadings.  This one could be more specific though and mention collaborative writing or related collaborative projects.

Reply:
We'll need at least one more pass through the discussion, so we can leave this for now.